### PR TITLE
Psp 6580 initiate filter screen for project exports

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

**Comment by devinleighsmith** [comment]
image

File: source/backend/dal/Repositories/AcquisitionFilePropertyRepository.cs [hunk 1]
```diff
@@ -52,22 +52,6 @@ public List<PimsPropertyAcquisitionFile> GetPropertiesByAcquisitionFileId(long a
...
-                .Include(x => x.Address)
-                    .ThenInclude(x => x.Country)
-                .Include(x => x.Address)
-                    .ThenInclude(x => x.ProvinceState)
-                .Include(x => x.Address)
-                    .ThenInclude(x => x.DistrictCodeNavigation)
-                .AsNoTracking()
-                .ToList();
-        }
-
```
Comment (asanchezr) [inline source/backend/dal/Repositories/AcquisitionFilePropertyRepository.cs:70]: Is this related to this pr or general code cleanup?

File: source/frontend/src/features/projects/reports/ProjectExportContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import { useProjectProvider } from '@/hooks/repositories/useProjectProvider';
+import { Api_ExportProjectFilter } from '@/models/api/ProjectFilter';
+
+import { IProjectExportFormProps } from './ProjectExportForm';
+
+export interface ISideProjectContainerProps {
+  View: React.FunctionComponent<React.PropsWithChildren<IProjectExportFormProps>>;
+}
+
+export const SideProjectContainer: React.FunctionComponent<ISideProjectContainerProps> = ({
```
Comment (asanchezr) [inline source/frontend/src/features/projects/reports/ProjectExportContainer.tsx:14]: nit: mild preference to have class name match filename ProjectExportContainer vs SideProjectContainer

File: source/frontend/src/features/projects/reports/ProjectExportForm.test.tsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    } as any);
+
+    const select = getByDisplayValue(/Select Export Type.../i);
+
+    await act(async () => {
+      userEvent.selectOptions(select, ProjectExportTypes.AGREEMENT);
+    });
+
+    expect(screen.getByText(/first last/g)).not.toBeNull();
+  });
```
Comment (asanchezr) [inline source/frontend/src/features/projects/reports/ProjectExportForm.test.tsx:110]: please remove duplicate test

File: source/frontend/src/features/projects/reports/ProjectExportForm.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+import { Api_Person } from '@/models/api/Person';
+import { Api_Project } from '@/models/api/Project';
+import { Api_ExportProjectFilter } from '@/models/api/ProjectFilter';
+
+import { ExportProjectModel, ProjectExportTypes } from './models';
+import { ProjectExportFormContent } from './ProjectExportFormContent';
+
+export interface IProjectExportFormProps {
+  onExportTypeSelected: () => void;
+  onExport: (filter: Api_ExportProjectFilter) => void;
```
Comment (asanchezr) [inline source/frontend/src/features/projects/reports/ProjectExportForm.tsx:15]: nit: we might update this later but likely the onExport should be an async function

**Review by asanchezr — APPROVED** [review]
